src/DotNet.Status.Web/Controllers/AnnotationsController.cs

```diff
@@ -0,0 +1,109 @@
+using Azure.Data.Tables;
+using DotNet.Status.Web.Models;
+using DotNet.Status.Web.Options;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.DotNet.Services.Utility;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace DotNet.Status.Web.Controllers
+{
+    /// <summary>
+    /// This Annotations controller serves queries in a format compatible with the Grafana
+    /// "Simple JSON Datasource". It is used to expose the "Deployments" table data to 
+    /// Grafana, which may then render the information in dashboards.
+    /// </summary>
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AnnotationsController : ControllerBase
+    {
+        private const int _maximumServerCount = 10; // Safety limit on query complexity
+        private readonly ILogger<AnnotationsController> _logger;
+        private readonly IOptionsMonitor<AnnotationsOptions> _options;
+
+        public AnnotationsController(ILogger<AnnotationsController> logger, IOptionsMonitor<AnnotationsOptions> options)
+        {
+            _logger = logger;
+            _options = options;
+        }
+
+        [HttpGet]
+        public ActionResult Get()
+        {
+            return Ok();
```

Comment:
Is "Ok" required? Or is a "NoContent" valid?

Reply:
I'll check. Is "NoContent" preferred?

Reply:
It's more HTTPly correct.  It's "success, but there isn't a body here, so you don't need to bother trying to read it".  It's unlikely anything would _break_, since 204 and 200 are genernally treated the same by everything. It's a good signal to the other other end that no, you aren't just being slow, and you didn't lose the body, there just isn't one.